configuration/pih/htmlforms/hiv/hiv-dispensing.xml

```diff
@@ -527,7 +527,27 @@
                 </tr>
             </table>
         </div>
+          <!-- Side effects or allergies: -->
+          <div class="section-container">
+            <table>
+                <tr>
+                    <td>
+                        <label><uimessage code="zl.allergy.drugQuestion"/></label>
+                        <br/>
+                        <obs id="drug_allergy_question"   conceptId="CIEL:165273" style="radio"/>
```

Comment:
For reference, here's the concept used: https://app.openconceptlab.org/#/orgs/PIH/sources/PIH/concepts/12352/